org.eclipse.lsp4j/src/test/java/org/eclipse/lsp4j/test/services/JsonParseTest.xtend

```diff
@@ -1417,4 +1422,76 @@ class JsonParseTest {
 			]
 		])
 	}
+	
+	
+	
+	@Test
+	def void testProgressCreate() {
+		'''
+			{
+				"jsonrpc": "2.0",
+				"id": 1,
+				"method": "window/workDoneProgress/create",
+				"params": {
+					"token": "progress-token"
```

Comment:
Please add tests with tokens that are numbers.

Reply:
ok